src/app/components/Results/ResultItems.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { isEmpty as _isEmpty } from 'underscore';
+
+const MAXDISPLAY = 5;
+
+class ResultsItems extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      collapsed: true,
+    };
+  }
+
+  getItems(items) {
+    return items.map((item, i) => {
+      const status = item.status;
+      const availability = item.availability;
+      const available = item.available;
+      const id = item.id;
+      const collapsed = this.state.collapsed;
+
+      return (
+        <li key={i}>
+          <div className={`sub-item ${i >= MAXDISPLAY && collapsed ? 'more' : ''}`}>
+            <div>
+              <span className={`status ${availability}`}>{status}</span>
+              {
+                available ? ' to use in ' : ' at location '
+              }
+              <span>{item.location}</span>
+              {
+                item.callNumber.length ?
+                (<span className="call-no"> with call no. {item.callNumber}</span>)
+                : null
+              }
+            </div>
+            <div>
+              {
+                item.url && item.url.length ?
+                (
+                  <a
+                    href={item.url}
+                    className="button"
+                  >
+                    {item.actionLabel}
+                    <span className="visuallyHidden"> {item.actionLabelHelper}</span>
+                  </a>
+                )
+                : null
+              }
+            </div>
+          </div>
+        </li>
+      );
+    });
+  }
+
+  getMoreLink(items) {
+    const itemCount = items.length;
+    const moreCount = itemCount - MAXDISPLAY;
+    const collapsed = this.state.collapsed;
+    const resultTitle = this.props.itemTitle;
+
+    return (
+      moreCount > 0 && collapsed ?
+      (
+        <li>
+          <a
```

Comment:
This should eventually be a button because it's not a link but an interactive element on the page. I had styling trouble so I left it as an `a` element for now but will eventually update this.